### PR TITLE
Chemical Revolution: Chemistry Skill and Lithium

### PR DESCRIPTION
**Pull request by roaringjohn**
### Summary
```SUMMARY: Content "Separate Chemistry from Cooking and add Lithium reaction recipes"```

#### Purpose of change
To start the expansion of chemistry in Cataclysm.

#### Describe the solution
As the base of any and all Chemical Revolution PRs I make, I separated the cooking skill into cooking and chemistry and plan to add lithium and lithium reactions as the first step.

#### Describe alternatives you've considered
For the skill separation? Proficiencies as mentioned in the Practice makes Perfect project. For the lithium stuff? I dunno.

#### Testing
Find lithium in labs, on scientists, etc. Make chemistry recipes to advance chemistry skill.

#### Additional context
First part of my "Chemical Revolution" series of PRs.

### CODE REVIEW

**Comment by NastyNate2612** [comment]
Lol i just put out my extracts and concentrates data the other day. whenever this gets added ill have to go and change the skill requirements for all the recipes for it

File: data/json/items/book/chemistry.json [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    "symbol": "?",
+    "color": "light_green",
+    "intelligence": 4,
+    "time": "30 m",
+    "fun": -2
+  },
+  {
+    "id": "salt_chemistry",
+    "type": "BOOK",
+    "name": { "str": "Chemistry of Metallic Salts", "str_pl": "copies of Chemistry of Metallic Salts" },
```
Comment (tenmillimaster) [inline data/json/items/book/chemistry.json]: This isn't what we'd call this field, really, unless it was really novel (Molten salt chemistry, for example).

Perhaps describe it as an inorganic chemistry textbook, with an included lab manual section?

Comment (roaringjohn) [reply data/json/items/book/chemistry.json]: Done. Probably gonna have to look into adding the book to more things, then.

**Comment by roaringjohn** [comment]
For personal reference after I return on Sunday:
Search through the recipes for tools that lean towards a chemistry focus to apply the chemistry skill to it if necessary.

**Comment by roaringjohn** [comment]
> Lol i just put out my extracts and concentrates data the other day. whenever this gets added ill have to go and change the skill requirements for all the recipes for it

If your PRs get merged first, I'll make sure to add to this PR to take your changes into account.

**Comment by ZhilkinSerg** [comment]
Feel free to reopen if you want working on it again.

**Pull request closed without merging**